cli/azd/docs/environment-variables.md

```diff
@@ -0,0 +1,19 @@
+# Environment Variables
```

Comment:
with expandableString, some of the fields from azure.yaml supports references to system env vars. Should we add a section about it and if it requires a prefix or anything?  @wbreza

Reply:
Any expandable string could reference system, shell or azd env vars.  Are you thinking of something specific that we should be calling out?

Reply:
I think in @vhvb1989 's mind we want the list of fields documented here.

If we ever get a user ask for this, let's do it. I think we're okay doing this later since we have self-describing help from our JSONSchema that is used for in-editor intellisense currently.